app/matplotlib.ipynb

```diff
@@ -194,21 +262,33 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 10,
-   "outputs": [],
    "source": [
     "labels = [\"Product A\", \"Product B\", \"Product C\", \"Product D\"]\n",
     "market_share = [30, 20, 15, 35]\n",
     "\n",
-    "# write your code here"
+    "plt.pie(market_share, labels=labels, autopct=\"%1.1f%%\", startangle=90)\n",
+    "plt.show()"
```

Comment:
The pie plot task requires setting a title for the plot. You can add a title using `plt.title("Market Share Distribution")` before `plt.show()`.